todo-winjs/WinJSTodo/scripts/index.ts

```diff
@@ -1,19 +1,34 @@
-﻿/***
-    Copyright (c) Microsoft. All rights reserved.  Licensed under the MIT license. See LICENSE file in the project root for full license information.
-*/
+﻿// For an introduction to the Blank template, see the following documentation:
+// http://go.microsoft.com/fwlink/?LinkID=397705
+// To debug code on page load in Ripple or on Android devices/emulators: launch your app, set breakpoints, 
+// and then run "window.location.reload()" in the JavaScript Console.
+module Todo {
+    "use strict";
 
-///<reference path='..\..\packages\cordova.TypeScript.DefinitelyTyped.2.3.3\Content\Scripts\typings\cordova\cordova.d.ts' />
-///<reference path='..\..\packages\winjs.TypeScript.DefinitelyTyped.0.1.9\Content\Scripts\typings\winjs\winjs.d.ts'/>
-///<reference path='..\..\packages\azure-mobile-services-client.TypeScript.DefinitelyTyped.0.8.6\Content\Scripts\typings\azure-mobile-services-client\AzureMobileServicesClient.d.ts'/>
-/// <reference path="services.ts" />
-/// <reference path="todo.ts" />
```

Comment:
Question: How does the TS compiler then knows about WinJS? If I remove the reference path in my project, it complains about: ` error TS2304: Build: Cannot find name 'WinJS'.`